internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
+
+type Configuration struct {
+	GithubToken string `env:"GITHUB_TOKEN,notEmpty"`
+	OpenAIToken string `env:"OPENAI_API_KEY,notEmpty"`
+}
+
+func LoadConfiguration() (Configuration, error) {
+	config := Configuration{}
```

Comment:
Consider handling the case when environment variables are not set or are empty. The `notEmpty` tag will cause `env.Parse` to return an error, but it would be beneficial to provide more context in the error message or handle the error more gracefully.